golang/Dockerfile.test

```diff
@@ -0,0 +1,8 @@
+FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:alpine AS builder

```

Comment:
The defaults seem unnecessary in here and in RUN.

Reply:
Should be ok now